Assets/Placenote/Toolkits/SpatialCapture/PlacenoteWebAPI.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+#if UNITY_EDITOR
+using UnityEditor;
+
+
+namespace EditorCoroutines {
```

Comment:
is this namespace needed ? can we use spatialcapture or do all editor coroutines need to be in that namespace ? also, how come this coroutine is here but the other editor coroutines are in the editor folder? does that make sense?

Reply:
EditorCoroutines need that namespace